problem/templates/problem/panel.html

```diff
@@ -22,14 +22,22 @@
 SOFTWARE.
 -->
 {% extends 'problem/base.html' %}
+{% load static %}
+{% block title_name %}
+<title>Problem Panel</title>
+{% endblock title_name %}
 {% block body_block %}
-<div class="container" id="Body">
-  <h2>Problem Panel</h2>
+<div class="container" id="list">
+  <br>
+  <!--title-->
+  <div class="well">
+    <h2>Problem Panel</h2>
+  </div>
   <div>
     {% if can_add_problem %}
-    <button id="add_problem_btn" style='display:inline-block' class="btn btn-warning" href="{% url 'problem:new' %}"
-      data-toggle="tooltip" data-placement="top" title="add problem">+</button>
-    <div id="add_problem_form" style="display:inline-block;margin-left:10px">
+    <button id="add_problem_btn" class="btn btn-warning" href="{% url 'problem:new' %}"
+      data-toggle="tooltip" data-placement="top" title="add problem">Add problem</button>

```

Comment:
remove title because that is not in HTML spec.
http://www.w3schools.com/tags/tag_button.asp